plugins/cue_points/quiz/lib/api/KalturaAnswerCuePoint.php

```diff
@@ -31,13 +31,6 @@ class KalturaAnswerCuePoint extends KalturaCuePoint
 	public $isCorrect;
 
 	/**
-	 * Array of string
```

Comment:
Since this parameters are part of the interfaces, It may be impossible to change them if they are already been used by application. Please verify with application team that $correctAnswerKeys is not being used already.